### ISSUE TITLE
Could not fire plugin

### ISSUE THREAD

**Issue by UnnamedSkill** [Zerthox/BetterDiscord-Plugins#195]
### Bug description

I'm unable to launch the plugin, tried downloading, re-downloading-deleting and pretty much everything
![image](https://github.com/user-attachments/assets/05dd8ca8-42f4-4750-921b-223451356486)

### Plugin

BetterVolume

### Plugin version

v3.1.0 (also on 3.0.0)

### Discord client

Stable

**Comment by Zerthox** [comment]
See #190, #193. The plugin's settings file `BetterVolume.config.json` was corrupted. Considering you are the 3rd user reporting this issue, I wonder what is causing this. Potentially a bug in BetterDiscord that is only triggered by BetterVolume...? Would it maybe be possible to send me your settings file?

**Comment by UnnamedSkill** [comment]
> See #190, #193. The plugin's settings file `BetterVolume.config.json` was corrupted. Considering you are the 3rd user reporting this issue, I wonder what is causing this. Potentially a bug in BetterDiscord that is only triggered by BetterVolume...? Would it maybe be possible to send me your settings file?

Sure where would you like me to send it

**Comment by kassake** [comment]
I am having the same issue, if it would help i can send show mine to compare.

**Comment by Zerthox** [comment]
You can upload the file here or send it via DM on Discord. Since v3.0.0 it will contain user ids for your volumes above 200%. For older versions it doesnt contain any data that could be considered sensitive.

**Comment by Raxgithub836** [comment]
Hey, I got this issue as well, managed to fix it by deleting/moving my config folder out. Here is/was the bugged folder
[BetterVolume.config.json](https://github.com/user-attachments/files/18007943/BetterVolume.config.json)

**Comment by Zerthox** [comment]
Yeah, the file got corrupted. It's full of null bytes. Not sure how that can happen. Did you do anything special before the plugin refused to start? Or do anything outside of Discord/BetterDiscord that could have affected the file?

**Comment by kassake** [comment]
> You can upload the file here or send it via DM on Discord. Since v3.0.0 it will contain user ids for your volumes above 200%. For older versions it doesnt contain any data that could be considered sensitive.

Here's mine
[BetterVolume.config.json](https://github.com/user-attachments/files/18014066/BetterVolume.config.json)

**Comment by Raxgithub836** [comment]
> Yeah, the file got corrupted. It's full of null bytes. Not sure how that can happen. Did you do anything special before the plugin refused to start? Or do anything outside of Discord/BetterDiscord that could have affected the file?

I had quite a few users above 200%, some of them I put to 400-450% ish because their mic volumes were low. 

Also, sometimes vanilla Discord would launch instead of BetterDiscord, which put those users down to 0%, to which I then adjusted their volumes just using the original slider. I would then reinstall/repair BetterDiscord after the fact which restored my original configs.

I did this process quite a few times, and it only started bugging out after the newest update.
